client/editSchedule.js

```diff
@@ -261,6 +313,27 @@ Template.editSchedule.rendered = function() {
   setupCompetitionCalendar(template, $calendar, $addEditSomethingModal);
 };
 
+Template.unscheduledRound.rendered = function() {
+  var template = this;
+  var $unscheduledRound = template.$('.fc-event');
+
+  // store data so the calendar knows to render an event upon drop
+  // $unscheduledRound.data('event');
+
+  // make the event draggable using jQuery UI
+  $unscheduledRound.draggable({
+    zIndex: 999,
+    revert: false,      // will cause the event to go back to its
+    revertDuration: 0  //  original position after the drag
+  });
+};
+
+Template.unscheduledRound.helpers({
+    title: function() {
+      return wca.eventByCode[this.eventCode].name + ": " + wca.roundByCode[this.roundCode].name;
```

Comment:
Is this logic still in multiple places? Perhaps we could consolidate it in helpers.js?